Examples/Volume/VolumeMapperBlendModes/index.js

```diff
@@ -50,13 +52,54 @@ ofun.addPoint(-637.62, 0.1);
 ofun.addPoint(700, 0.5);
 ofun.addPoint(3071, 0.9);
 
-const ctfun = vtkColorTransferFunction.newInstance();
-ctfun.addRGBPoint(-3024, 0, 0, 0);
-ctfun.addRGBPoint(-637.62, 0, 0, 0);
-ctfun.addRGBPoint(700, 1, 1, 1);
-ctfun.addRGBPoint(3071, 1, 1, 1);
+const radonParameters = {
+  min: 1530,
+  max: 1030,
+  maxValue: 0.017,
+  minValue: 0.092,
+  unit: 0.3,
+};
+
+function updateColorFunction() {
+  const ctfun = vtkColorTransferFunction.newInstance();
+
+  if (currentBlendMode === 5) {
+    ctfun.addRGBPoint(-3024, 0.0, 0.0, 0.0);
+    ctfun.addRGBPoint(
+      radonParameters.min,
+      radonParameters.minValue,
+      radonParameters.minValue,
+      radonParameters.minValue
+    );
+    ctfun.addRGBPoint(
+      radonParameters.max,
+      radonParameters.maxValue,
+      radonParameters.maxValue,
+      radonParameters.maxValue
+    );
+    ctfun.addRGBPoint(
+      3071,
+      radonParameters.maxValue,
+      radonParameters.maxValue,
+      radonParameters.maxValue
+    );
+
+    minLabel.innerHTML = radonParameters.min;
+    maxLabel.innerHTML = radonParameters.max;
+    maxValueLabel.innerHTML = radonParameters.maxValue;
+    minValueLabel.innerHTML = radonParameters.minValue;
+    unitLabel.innerHTML = radonParameters.unit;
+    mapper.setSampleDistance(radonParameters.unit);
```

Comment:
`unit` does not seem proper to radon. Move it outside radon ?

Reply:
This is not proper to radon, but it wasn't exposed on the other blend type in the initial example. I don't want to change the behavior of it. So I will reinitialize sample distance to initial value when setting blend mode other than radon.

Reply:
Let's expose it to all blend modes to see the impact of the sample distance on each blend mode